sdks/java/harness/build.gradle

```diff
@@ -63,4 +63,5 @@ dependencies {
 // dependencies without repackaging.
 shadowJar {
   mergeServiceFiles()
```

Comment:
We should change the rule in the docker build to match the updated names instead of doing it here.

Better yet if we could grab the "shadow" configuration artifact from this project regardless of its name inside the docker build for the container.

Reply:
Done. I couldn't figure out how to get the shadow jar artifact name directly from gradle.

Reply:
Maybe another day.